Cargo.toml

```
@@ -69,8 +69,8 @@ rayon = "1.5"
 
 # Metrics
 ipnet = "2"
-metrics = "0.17"
-metrics-exporter-prometheus = "0.6"
+metrics = "0.19"
+metrics-exporter-prometheus = { version = "0.10", features = ["http-listener"] }
```

Currently this does nothing, because Cargo features are purely additive. Either add `default-features = false`:
```suggestion
metrics-exporter-prometheus = { version = "0.10", default-features = false, features = ["http-listener"] }
```

or remove the `features` arg:
```suggestion
metrics-exporter-prometheus = "0.10"
```

---

I chose the latter.